react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js

```diff
@@ -180,7 +189,22 @@ class ChromeExtensionBanner extends PureComponent<Props, State> {
 
         return dontShowAgain
         || this.state.closePressed
-        || !this.state.shouldShow;
+        || !this.state.shouldShow
+        || this.props.iAmRecorder;
+    }
+
+    /**
+     * Decides whether the chrome extension banner is enabled based on config and environment.
+     *
+     * @returns {boolean}
+     * @protected
+     */
+    _isBannerEnabled() {
```

Comment:
Why are half of the checks on this function and the other half on _shouldNotRender? I'd say let's move all of them to a single place.

Reply:
unified the check in a single place